pkgs/by-name/co/coroot-node-agent/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  systemdLibs,
+}:
+
+buildGoModule rec {
+  pname = "coroot-node-agent";
+  version = "1.21.9";
+
+  src = fetchFromGitHub {
+    owner = "coroot";
+    repo = "coroot-node-agent";
+    rev = "v${version}";
+    hash = "sha256-gnGYZtH/WGhGL6ehwiu6cHc4scG9mrRcfVzRaijLrWU=";
+  };
+
+  vendorHash = "sha256-OZj3t8eFDbk2zZ+zGlgV8o9VC4/vYalIEm2dhD2JwK8=";
+
+  nativeBuildInputs = [ ];
+  buildInputs = [ systemdLibs ];
+
+  CGO_CFLAGS = "-I ${systemdLibs}/include";
+
+  meta = {
+    description = "Prometheus exporter based on eBPF";
```

Comment:
My understanding, based on https://github.com/NixOS/nixpkgs/tree/master/pkgs#category-hierarchy is that (almost) everything lives under `by-name` these days. Another reason for doing this is that while functionality-wise the developers decided to describe this as a prometheus exporter, it's mainly tied to the `coroot` package instead of prometheus.

Reply:
You are right, https://github.com/NixOS/nixpkgs/blob/master/nixos%2Fmodules%2Fservices%2Fmonitoring%2Fprometheus%2Fexporters.md doesn't say anything about that. Then it is okay